src/Facade/Services/TariffService.cs

```diff
@@ -1,77 +1,37 @@
-﻿namespace Linn.Products.Facade.Services
-{
-    using System;
-    using System.Collections.Generic;
-    using Linn.Common.Facade;
-    using Linn.Common.Persistence;
-    using Linn.Products.Domain.Linnapps.Products;
-    using Linn.Products.Domain.Linnapps.Repositories;
-    using Linn.Products.Resources;
+﻿using System;
+using Linn.Common.Facade;
+using Linn.Common.Persistence;
+using Linn.Products.Domain.Linnapps.Products;
+using Linn.Products.Resources;
```

Comment:
These usings should be down a level, this might just be a resharper setting.

Reply:
👍